extensions/ql-vscode/src/helpers.ts

```diff
@@ -785,3 +785,24 @@ export async function* walkDirectory(
     }
   }
 }
+
+export function getFirstStoragePath() {
+  const workspaceFolders = workspace.workspaceFolders;
```

Comment:
Can we use `getOnDiskWorkspaceFolders` here instead of `workspace.workspaceFolders`? That will filter out any workspace folders which are not on the disk (e.g. databases).